Improve error message in IR parser when accessing undefined variable.

ZolotukhinM opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#26771 Improve error message in IR parser when accessing undefined variable.**

Differential Revision: [D17562853](https://our.internmc.facebook.com/intern/diff/D17562853)

Commit:
Improve error message in IR parser when accessing undefined variable.

[ghstack-poisoned]

jerryzh168 reviewed (approved):
LG

eellison commented on torch/csrc/jit/irparser.cpp:
```diff
@@ -255,7 +255,10 @@ void IRParser::parseOperatorInputs(Node* n) {
     std::string var_name = parseVar();
-    AT_ASSERT(vmap.count(var_name));
+    if (!vmap.count(var_name)) {
+      throw ErrorReport(L.cur().range)
```
Reduce copy - pasta ?

Commit:
Update on "Improve error message in IR parser when accessing undefined variable."


Differential Revision: [D17562853](https://our.internmc.facebook.com/intern/diff/D17562853)

[ghstack-poisoned]

Commit:
Rebase on "Improve error message in IR parser when accessing undefined variable."


Differential Revision: [D17562853](https://our.internmc.facebook.com/intern/diff/D17562853)

[ghstack-poisoned]
